src/resolvers/Query.js

```diff
@@ -98,10 +98,46 @@ function organizations(_, args, context, info){
   );
 }
 
+function approvals(_, args, context, info){
+  var outstandingApprovals = {};
+  var submittedApprovals = {};
+  
+  if (propertyExists(args, "gcIDApprover")){
+    outstandingApprovals = {
+      gcID: copyValueToObjectIfDefined(args.gcIDApprover.gcID),
+    };
+  }
+
+  if (propertyExists(args, "gcIDSubmitter")){
+    submittedApprovals = {
+      gcID: copyValueToObjectIfDefined(args.gcIDSubmitter.gcID),
+    };
+  }
+  return context.prisma.query.approvals(
+    {
+      where:{
+        id: copyValueToObjectIfDefined(args.id),
+        gcIDSubmitter: copyValueToObjectIfDefined(submittedApprovals),
+        gcIDApprover: copyValueToObjectIfDefined(outstandingApprovals),
+      
+        // eslint-disable-next-line camelcase
+        status: copyValueToObjectIfDefined(args.status),
+        // eslint-disable-next-line camelcase
+        changeType: copyValueToObjectIfDefined(args.changeType),
+
+      },
+      skip: copyValueToObjectIfDefined(args.skip),
```

Comment:
skip and first are declared here but not in schemal.graphql for the Query approvals